packages/apsara-ui/src/Listing/Listing.types.tsx

```diff
@@ -1,28 +1,32 @@
+import { SorterResult } from "../Table/TableProps";
 import { IVirtualTable } from "../Table/VirtualisedTable";
-export interface ListingProps {
-    list?: any[];
+import { Column } from "../TableV2/VirtualisedTable";
+
+export type ColumnRenderFunc<T> = (path: string, sortedInfo: SorterResult<T>) => Column<T>[];
+
+export interface ListingProps<T> {
+    list?: T[];
     loading?: boolean;
     resourceName?: string;
     resourcePath?: string;
     rowKey?: string;
     className?: string;
     tableProps?: {
-        getColumnList?: any;
-        handleRowClick?: (event: any, rowIndexData: any) => void;
+        getColumnList?: ColumnRenderFunc<T> | undefined;
```

Comment:
we can explicitly remove `| undefined`.

adding `?` suffix is equivalent to `| undefined`.